### ISSUE TITLE
Monorepo Changelogs

### ISSUE THREAD

**Issue by delmendo** [git-chglog/git-chglog#100]
I currently use chglog with a monorepo. This generates a Changelog at the top of the directory structure.

With a structure like

/backend-app
/mobile-app
/web-app
/website

I'd love to be able to have changelogs in each sub directory that only use the entries that are committed on files in the respective directories  For example
If I modify a file i /backend-app  with a feature:  record I'd like that changelog message to only show up in the changelog in /backend-app and not in any of the other changelogs.  Is anything like this on the road map?

**Comment by mavogel** [comment]
>  Is anything like this on the road map?

No, currently we did not plan that. Can you provide a demo repo that we could use as a baseline for testing/implementing this feature?

**Comment by clok** [comment]
I believe that feature PR #62 would help accomplish this.

**Comment by mavogel** [comment]
#62 got merged, let's verify if we can close this as well

**Comment by mavogel** [comment]
@delmendo can you re-check with the latest `v0.11.0` release?

**Comment by W1M0R** [comment]
> #62 got merged, let's verify if we can close this as well

This works for me. Thanks!
